udata_front/theme/gouvfr/assets/js/components/MultiSelect/MultiSelect.stories.js

```diff
@@ -14,10 +14,14 @@ const args = {
     {
       label: "Option 1",
       value: "Option 1",
+      helper: "Insee: 53350",
```

Comment:
You can make another `const` instead of adding these to the first one to have both case tested.